elements/rh-spinner/rh-spinner.css

```
@@ -0,0 +1,81 @@
+:host {
+  display: inline-block;
+  aspect-ratio: 1 / 1;
+  text-align: center;
+  --rh-spinner--AnimationDuration: 1.2s;
+  --rh-spinner--StrokeWidth: 6px;
+  /*--rh-spinner--AnimationTimingFunction: ;*/
+}
+
+svg {
+  width: var(--rh-spinner--width, var(--rh-spinner--diameter, var(--rh-length-4xl, 64px)));
```

```suggestion
  width: var(--rh-spinner-width, var(--rh-spinner-diameter, var(--rh-length-4xl, 64px)));
```

etc.

although tbh `--rh-spinner-width` might be superfluous. Why not just make the svg's width 100% and have users set built-in property `width` on the host?